schema/gene_families.js

```diff
@@ -1,11 +1,20 @@
 import GeneFamily from "./gene_family"
-import { GraphQLList } from "graphql"
+import { GraphQLList, GraphQLInt } from "graphql"
 
 const GeneFamilies = {
   type: new GraphQLList(GeneFamily.type),
   description: "A list of Gene Families",
-  resolve: (_source, _args, _request, { rootValue }) => {
-    return rootValue.geneFamiliesLoader()
+  args: {
+    size: {
+      type: GraphQLInt,
+    },
+    page: {
+      type: GraphQLInt,
+    },
+  },
+  resolve: (_source, options, _request, { rootValue }) => {
+    const gravityOptions = Object.assign({}, options, { sort: "position" })
```

Comment:
You're the only one using this query?

Reply:
Yep, for the new /categories page. 

Do you think this should be more generalized and actually take a `sort` arg in the query?

Reply:
Probably not going to get much value today for it, worth leaving until you need to support changing the sorts

Reply:
Yeah, any public facing consumer will want this default sort order. Will leave as-is then. 👍